client/src/pages/Feed.js

```diff
@@ -18,21 +30,151 @@ import {
 // ICONS
 import SvgIcon from "components/Icon/SvgIcon";
 import creatPost from "assets/icons/create-post.svg";
+import { ReactComponent as FiltersIcon } from "assets/icons/filters.svg";
 
 export const FeedContext = React.createContext();
 
+const { Content, Sider } = Layout;
+
+// feed types
+const HELP_TYPE = {
+  ALL: 'All posts',
+  REQUEST: 'Requesting help',
+  OFFER: 'Offering help'
+};
+
 const initialState = {
```

Comment:
Currently this component uses `useReducer` to handle local state. And I see that there's some `useState` logic introduced. Do you think we should stick with using just one hook for local state? I don't have a preference but it seems we should use `useState` since state isn't too complex.  I initially used `useReducer` because I thought state would be complex but it ended up rather basic.

Reply:
I will move everything to state, but I will leave commented on current calls if useReducer will be used later.

Reply:
Done.